lib/MySQL_Session.cpp

```diff
@@ -563,7 +563,6 @@ bool MySQL_Session::handler_special_queries(PtrSize_t *pkt) {
 			mysql_hdr Hdr;
 			memcpy(&Hdr,pkt->ptr,sizeof(mysql_hdr));
 			Hdr.pkt_length=pkt_2.size-5;
-			memcpy((char *)pkt_2.ptr+5,(char *)pkt->ptr+5,1);
```

Comment:
To be correct, this should actually be:
```
memcpy((char *)pkt_2.ptr+4,(char *)pkt->ptr+4,1);
```
Part of this code can be removed completely, as this only perform a transformation.
Nonetheless, I will fix it for correctness.